src/utils/@oasisprotocol/ledger.d.ts

```diff
@@ -0,0 +1,5 @@
+// TODO: remove and add types upstream
+declare module '@oasisprotocol/ledger' {
+  var x: any
+  export = x
+}
```

Comment:
TODO

Reply:
Are we waiting for this TODO to be resolved in this PR or in a following one? not sure this PR is ready for review or not 😄

Reply:
Oops :) Ready for review

I think adding types to that upstream library will take a bit

Reply:
Created https://github.com/oasisprotocol/ledger-js/pull/415